src/scenes/Fog.js

```diff
@@ -23,12 +23,16 @@ class Fog {
 
 	toJSON( /* meta */ ) {
 
-		return {
-			type: 'Fog',
-			color: this.color.getHex(),
-			near: this.near,
-			far: this.far
-		};
+		const data = {};
+
+		data.type = 'Fog';
+		data.color = this.color.getHex();
+
+		if ( this.name !== '' ) data.name = this.name;
+		if ( this.near !== 1 ) data.near = this.near;
+		if ( this.far !== 1000 ) data.far = this.far;
```

Comment:
Do you mind restoring the old style?

I know other classes do it differently but the problem with this approach is the redefinition of default values. Meaning default parameters should ideally defined just once in the ctor and not again at other places. The resulting JSON might hold a default value but I don't have the feeling the file size benefits outweighs the maintainability.

Reply:
get it.